test/CheckoutSdkTest/Disputes/Previous/DisputesIntegrationTest.cs

```diff
@@ -11,9 +11,10 @@ public class DisputesIntegrationTest : AbstractPaymentsIntegrationTest
         [Fact]
         private async Task ShouldQueryDisputes()
         {
+            //TODO format
```

Comment:
What about this TODO? can we solve it now?

Reply:
Sorry, mistake, I have deleted it

Reply:
Thank you!